composer.json

```diff
@@ -51,7 +51,8 @@
     },
     "conflict": {
         "container-interop/container-interop": "<1.2.0",
-        "zendframework/zend-expressive-zendviewrenderer": "*"
+        "zendframework/zend-expressive-zendviewrenderer": "*",
+        "zendframework/zend-view": "*"
```

Comment:
Can IMO be dropped - doesn't have any effect, other than cluttering the dependency resolution logic